doc/interop-test-descriptions.md

```diff
@@ -941,6 +962,17 @@ payload body of size `SimpleRequest.response_size` bytes and type as appropriate
 for the `SimpleRequest.response_type`. If the server does not support the
 `response_type`, then it should fail the RPC with `INVALID_ARGUMENT`.
 
+### CacheableUnaryCall
+
+Server gets the default SimpleRequest proto as the request. The content of the
+request are ignored. It returns the SimpleResponse proto with the payload set 
+to current timestamp string.  In addition it adds
+  1. cache control headers such that the response can be cached by proxies in
```

Comment:
Specify a minimum time for the request to be cached. I'd also suggest providing an example/encouraged cache control string.